src/main/java/com/quadfour/QuadFourController.java

```diff
@@ -78,7 +78,7 @@ public String task(@RequestParam(value="taskId") Integer taskId, Model model) {
 
     	// TODO: add in route/behavior for "/task" without "taskId" parameter
 
-    	TaskDTO taskDTO = taskService.fetchById(taskId);
+    	TaskDTO taskDTO = taskService.fetchTaskById(taskId);
```

Comment:
It's safe to leave this as is.  The fact that it is a task ID is implicit, given the parameter taskId... and the return type TaskDTO, with a variable named taskDTO, which is invoked on taskService.  Adding task one more time is redundant.